content/docs/2.14/scalers/metrics-api.md

```diff
@@ -27,7 +28,10 @@ triggers:
 **Parameter list:**
 
 - `url` - Full URL of the API operation to call to get the metric value (eg. `http://app:1317/api/v1/stats`).
-- `valueLocation` - [GJSON path notation](https://github.com/tidwall/gjson#path-syntax) to refer to the field in the payload containing the metric value.
+- `format` - One of the following formats: `json`, `xml`, `yaml`, `prometheus`. (Default: `json`, Optional)
+- `valueLocation` - The location of the metric value in the response payload. The value is format specific.
+  * `json` - [GJSON path notation](https://github.com/tidwall/gjson#path-syntax) to refer to the field in the payload containing the metric value.
+  * `yaml`, `xml`, `prometheus` - We use internal implementation that retrieves a value using a dot-separated path: [#2633](https://github.com/kedacore/keda/issues/2633).
```

Comment:
Could you add an explanation about how it works please? the issue isn't clear enough